p2p/state.py

```diff
@@ -135,7 +136,14 @@ def msg_queue_maxsize(self) -> int:
                 # A node may be received more than once, so pop() with a default value.
                 self._pending_nodes.pop(node_key, None)
         elif isinstance(cmd, eth.GetBlockHeaders):
-            await self._handle_get_block_headers(peer, cast(Dict[str, Any], msg))
+            query = cast(Dict[Any, Union[bool, int]], msg)
```

Comment:
Why do we need to use `Any` for the dict's key here?

Reply:
No, will update to be `str`